types/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// TypeScript Version: 3.0
+
+import {Node} from 'unist'
+import stringifyEntities = require('stringify-entities')
+
+declare namespace hastUtilToHtml {
+  interface HastUtilToHtmlOptions {
+    /**
+     * Whether the *root* of the *tree* is in the `'html'` or `'svg'` space (enum, `'svg'` or `'html'`, default: `'html'`).
+     *
+     * If an `svg` element is found in the HTML space, `toHtml` automatically switches to the SVG space when entering the element, and switches back when exiting.
+     */
+    space: 'html' | 'svg'
+
+    /**
+     * Configuration for `stringify-entities` (`Object`, default: `{}`).
+     * Do not use `escapeOnly`, `attribute`, or `subset` (`toHtml` already passes those, so they won’t work).
```

Comment:
Might we want to `Omit<>` these from the type?
https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#the-omit-helper-type